### PR TITLE
Feature.303.short term history

### PR DESCRIPTION
**Pull request by jicarretero**

### CODE REVIEW

File: features/303.Short_term_history.feature [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    When  I prepare a GET HTTP request to "http://localhost:1026/v2/subscriptions/"
+    And   I set header fiware-service to openiot
+    And   I set header fiware-servicepath to /
+    And   I perform the query request
+    Then  I receive a HTTP "200" response code
+
+
+ # 4.5 - Request - Populate Lamp:001 history with curls
+ Scenario Outline: Communicating with IoT Devices: Using Actuators
+    # When  I prepare a POST HTTP request to "http://localhost:7896/iot/d?k=3314136&i=lamp001"
```
Comment (flopezag) [inline features/303.Short_term_history.feature]: maybe you can delete this comment.

File: features/steps/303.ShortTermHistory.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import time
+import jq
+
+from behave import given, step, when
+from os.path import join
+from config.settings import CODE_HOME
+
+
+@given(u'I set the tutorial 303')
+def step_impl_tutorial_205(context):
```
Comment (flopezag) [inline features/steps/303.ShortTermHistory.py]: step_impl_tutorial_303

File: features/steps/303.ShortTermHistory.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+def step_impl(context, sensor_values):
+    context.payload = sensor_values
+
+
+@step(u'I substitute in payload "{what}" for "{dst}"')
+def step_impl(context, what, dst):
+    f = context.payload
+    f = f.replace(what, dst)
+    context.payload = f
+    pass
```
Comment (flopezag) [inline features/steps/303.ShortTermHistory.py]: I think that it is not needed to call pass if the step_impl finished without problems.

File: features/steps/303.ShortTermHistory.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+@step(u'I validate against JQ {expr}')
+def step_impl(context, expr):
+    expr = expr.replace('#124;', '|')
+    pl = context.response
+    jqe = jq.compile(expr)
+
+    i = iter(jqe.input(pl))
+    r = next(i, None)
+    assert r == True
```
Comment (flopezag) [inline features/steps/303.ShortTermHistory.py]: add new line

**Review by flopezag — CHANGES_REQUESTED** [review]

File: features/steps/303.ShortTermHistory.py (hunk 3 above)
Comment (jicarretero) [reply features/steps/303.ShortTermHistory.py]: This function is no longer needed. I remove it.

Comment (flopezag) [reply features/steps/303.ShortTermHistory.py]: 👌

File: features/steps/303.ShortTermHistory.py (hunk 2 above)
Comment (flopezag) [reply features/steps/303.ShortTermHistory.py]: 👌

File: features/303.Short_term_history.feature (hunk 1 above)
Comment (flopezag) [reply features/303.Short_term_history.feature]: 👌

File: features/steps/303.ShortTermHistory.py (hunk 4 above)
Comment (flopezag) [reply features/steps/303.ShortTermHistory.py]: 👌

**Comment by flopezag** [comment]
LGTM

**Review by flopezag — APPROVED** [review]

**Pull request merged**